docsets/Vulkan/docset.json

```diff
@@ -1,10 +1,20 @@
 {
     "name": "Vulkan",
-    "version": "1.0.38",
+    "version": "1.0.39",
     "archive": "Vulkan.tgz",
```

Comment:
Sorry I don't quite understand. `docsets/Vulkan/versions/1.0.39/Vulkan.tgz` is already there.